content/en/about/feature-stages/index.md

```diff
@@ -55,12 +55,10 @@ Below is our list of existing features and their current phases. This informatio
 |-------------------|-------------------
 | [Prometheus Integration](/docs/tasks/observability/metrics/querying-metrics/) | Stable
 | [Local Logging (STDIO)](/docs/tasks/observability/mixer/logs/collecting-logs/) | Stable
-| [Statsd Integration](/docs/reference/config/policy-and-telemetry/adapters/statsd/) | Stable
 | [Client and Server Telemetry Reporting](/docs/reference/config/policy-and-telemetry/) | Stable
 | [Service Dashboard in Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/) | Stable
 | [Istio Component Dashboard in Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/) | Stable
 | [Distributed Tracing](/docs/tasks/observability/distributed-tracing/) | Stable
-| [Stackdriver Integration](/docs/reference/config/policy-and-telemetry/adapters/stackdriver/) | Beta
```

Comment:
this should be kept, but maybe file a bug to provide better docs?

Reply:
The referenced docs, mixer/adapter/stackdriver/config/adapter.stackdriver.config.pb.html, were moved in the big removal PR, https://github.com/istio/istio/pull/25669. https://github.com/istio/istio/issues/26993 was created for a new doc to reference.

Reply:
Also made `Stable`